workflow-service/src/main/java/com/redhat/parodos/security/SecurityConfiguration.java

```diff
@@ -45,19 +44,20 @@
 @Configuration
 @Profile("!local")
 @DependsOn("org.springframework.boot.context.properties.ConfigurationPropertiesBindingPostProcessor")
-public class SecurityConfiguration {
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
```

Comment:
`WebSecurityConfigurerAdapter` is deprecated: `'org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter' is deprecated ` so we should not be using it
Instead we should use `AuthenticationManager`  'ie: https://docs.spring.io/spring-security/reference/servlet/authentication/passwords/ldap.html#servlet-authentication-ldap-pwd and https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter)